js/ui/text_box/ui.text_editor.base.js

```diff
@@ -432,6 +443,27 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
+    _renderLabel: function(options) {
+        if(this._$label) {
+            this._$label.remove();
+            this._$label = null;
+        }
+
+        this.$element().removeClass(TEXTEDITOR_WITH_LABEL_CLASS);
+        this.$element().removeClass(TEXTEDITOR_WITH_FLOATING_LABEL_CLASS);
+
+        if(!this.option('label') || this.option('labelMode') === 'hidden') return;
+
+        this.$element().addClass(this.option('labelMode') === 'floating' ? TEXTEDITOR_WITH_FLOATING_LABEL_CLASS : TEXTEDITOR_WITH_LABEL_CLASS);
+
+        const labelText = this.option('label');
+        const $label = this._$label = $('<div>')
```

Comment:
const $labelText = $('<span>').text(this.option('label'));
this._$label = $('<div>')
    .addClass(TEXTEDITOR_LABEL_CLASS)
    .append($labelText)

this._$label.appendTo(this._$textEditorInputContainer);

?